src/plugins/react/reactquestiondate.tsx

```diff
@@ -24,10 +24,14 @@ export default class SurveyQuestionDate extends React.Component<any, any> {
     }
     render(): JSX.Element {
         if (!this.question) return null;
-        var date = this.question.value ? moment(this.question.value, "DD/MM/YYYY") : moment();
+        var value = this.question.value;
         return (
             <div id={this.getDivId()}>
-                <DatePicker className="form-control" showYearDropdown selected={date} onChange={this.handleOnChange} />
+                <Cleave placeholder="DD/MM/YYYY"
+                className="form-control"
```

Comment:
Agreed. If design / product wants this I'm all for going with the simpler component, but since it is losing the graphical date picker functionality it's probably their call.